src/range/rangeSlider.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useCallback, useEffect, useRef } from "react";
+import style from './rangeSlider.module.css';
+
+const RangeSlider = ({ min, max, onChange, value }) => {
+
+    const range = useRef(null);
+
+    const getPercent = useCallback(
+        (value) => Math.round(((value - min) / (max - min)) * 100),
+        [min, max]
+    );
+
+    useEffect(() => {
+        const minPercent = getPercent(min);
+        const maxPercent = getPercent(value);
+
+        if (range.current) {
+            range.current.style.width = `${maxPercent - minPercent}%`;
+        }
+    }, [value, getPercent]);
```

Comment:
why don't you add `min` and `range` to the list of your dependencies? please check everywhere else that you correctly provide dependencies to your hooks